diagnostics/model_test/README.md

```diff
@@ -1,21 +1,24 @@
-# Compare model output between Tensorflow and NGraph
+# Compare model output between two different backends
 
-### This model_test tool will run the model inference seperately on TF and NGraph, and the desired output from TF and NGraph should match given the same inputs. It can be used as a debugging tool, and also a verification that NGraph produces the same output as Tensorflow. 
+### This model_test tool will run the model inference seperately on two specified backends(e.g.Tensorflow and nGraph) in json file and the desired outputs from each backend should match given the same inputs. It can be used as a debugging tool for layer by layer comparison, and also a verification that nGraph produces the same output as Tensorflow. 
 
 # Required files to use the tool:
-* A json file: Provide model specific parameters. Look at the example ```mnist_cnn.json```. You can start with the ```template.json``` and modify it to match your model
+* A json file: Provide model specific parameters. Look at the example ```mnist_cnn.json```.
 * A tensorflow frozen graph: A model frozen graph(.pb file) with trained weights and model architecture
```

Comment:
With your change, we can also have a model .pbtxt file without the weights right? So user can supply either a tensorflow frozen graph, or a tensorflow model .pbtxt file.